stdlib/experimental/http/http.flux

```diff
@@ -1,6 +1,34 @@
+// Package http provides functions for transferring data using HTTP protocol.
+// 
+// introduced: 0.39.0
+// tags: http
+// 
 package http
 
 
-// Get submits an HTTP get request to the specified URL with headers
-// Returns HTTP status code and body as a byte array
+// get submits an HTTP GET request to the specified URL and returns the HTTP
+// status code, response body, and response headers.
+// 
+// ## Response format
+// `http.get()` returns a record with the following properties:
+// 
+// - **statusCode**: HTTP status code returned by the GET request (int).
+// - **body**: HTTP response body (bytes). 
+// - **headers**: HTTP response headers (record).
+// 
+// ## Parameters
+// - url: URL to send the GET request to.
+// - headers: Headers to include with the GET request.
```

Comment:
Will it be clear to the reader that headers is an object with like `{Authorization: "foo", Connection: "keep-alive"}`? Might be good to include an example. (Similar comment for the other functions that have a parameter like this)

Reply:
Added headers to the example below.